src/index.js

```diff
@@ -1,17 +1,36 @@
 import _ from 'lodash';
 import './css/main.scss';
+import cities from 'cities.json';
+import nations from 'countries-list';
+import DOMController from './DomStuff';
+import weatherConditionGifs from './data';
+
 
 const getWeather = (cityID) => {
-	const apiKey = '612ba83bcef55aa96b88795e503e2ec8';
-	fetch('http://api.openweathermap.org/data/2.5/weather?q=London,uk&APPID=612ba83bcef55aa96b88795e503e2ec8', {
-			mode: 'cors'
-		})
-		.then((response) => {
-			console.log(response.json());
-		})
-		.catch((err) => {
-			console.log(err);
-		});
+  const apiKey = '612ba83bcef55aa96b88795e503e2ec8';
+  fetch(`https://api.openweathermap.org/data/2.5/weather?q=${cityID}&APPID=${apiKey}`, {
+    mode: 'cors',
+  }).then((response) => response.json())
+    .then((data) => {
+      const source = `https://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png`;
+      fetch(`https://api.giphy.com/v1/gifs/${weatherConditionGifs[`${data.weather[0].main}-${data.weather[0].icon[2]}`]}?api_key=FnaO3rXcDZNixxotdnLqLtYDrJYztxa1`)
```

Comment:
there is too much going on here. let our data fetch and data processing  be separated.
here is what i suggest. when `getWeather` is called, it should  make and async call to a method that will just fetch the data and send it to `getWeather` and it can send it to the DOM. that way we can change our implementation of data fetch without affecting the rest of the app.

- lets do better error handling, we have strong requirement to avoid having console.log in our code. let's clean that out too.